### PR TITLE
HPCC-29356 add JIRA creation guidelines to userdocs

### PR DESCRIPTION
**Pull request by cloLN**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [x] Security
  - [x] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by stuartort — APPROVED** [review]
This addition looks fine

File: devdoc/userdoc/JiraGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - These include input files, code, pkgmap contents, etc.
+   - Type of environment where the HPCC Platform is running.
+      - Bare-metal environment, VM, container?
+      - Details on the systems used to start these such as:
+         - Operation system 
+         - Minikube or Docker Desktop
+         - Virtual Box
+   - Steps taken to install the platform if this is related to installation and init.
+   - If this is a bare-metal build, where did you get your package and what is the md5sum of this package?
+   - Include stack traces and a copy of the core file if possible.
```
Comment (ghalliday) [inline devdoc/userdoc/JiraGuidelines.md:27]: I don't think we generally want core files attached - they may contain pii, and can be GBs in size.

File: devdoc/userdoc/JiraGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+This is a guide to the type of information that is desired in JIRA tickets.
+
+# For bugs / improvement requests on any existing features or docs: 
+   - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
+      - This is especially true for issues related to ECL Watch.
+   - Provide a zap file if there is one.
+   - Build version of the environment you're using.
+   - URL to environment.
+   - URL to workunit.
+   - Which browser and its version.
```
Comment (ghalliday) [inline devdoc/userdoc/JiraGuidelines.md:12]: would it be worth grouping this with screenshot under user-interface issues?

**Review by ghalliday — COMMENTED** [review]
Looks good.  Please remove adding core files.  I'm happy for you to leave the rest as it is.

**Comment by ghalliday** [comment]
@cloLN this is waiting for you to update it...

File: devdoc/userdoc/JiraGuidelines.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
 This is a guide to the type of information that is desired in JIRA tickets.
 
 # For bugs / improvement requests on any existing features or docs: 
-   - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
-      - This is especially true for issues related to ECL Watch.
+   - User-Interface Issues:
+      - Provide a screenshot when it’s applicable.  Not just of the offending line or excerpt, but of the entire application window.
+      - URL to environment.
```
Comment (ghalliday) [inline devdoc/userdoc/JiraGuidelines.md:8]: url to environment and workunit are relevant to all issues.

Which browser and version is only relevant to UI.

**Comment by ghalliday** [comment]
@cloLN this is still waiting for you...

File: devdoc/userdoc/JiraGuidelines.md (hunk 3 above)
Comment (cloLN) [reply devdoc/userdoc/JiraGuidelines.md]: I made another commit for this.  comment.  It should take care of it.

**Review by ghalliday — APPROVED** [review]
@cloLN thanks, please squash

**Review by ghalliday — CHANGES_REQUESTED** [review]
@cloLN your force push removed the last commit rather than squashing them.

**Comment by cloLN** [comment]
> @cloLN your force push removed the last commit rather than squashing them.

Oof.  Let me try again.

**Comment by cloLN** [comment]
I'm not quite sure how to roll back my changes.  Suggestions?  Should I try to revert changes in git or should I proceed with what I have here as it should fix all the concerns in the previous comments?

**Comment by cloLN** [comment]
This PR should probably be deleted and a new one be made for Github Issues.

**Comment by cloLN** [comment]
Will refactor for Github Issues
